stretchly-sources.yaml

```diff
@@ -0,0 +1,10 @@
+  - type: git
+    url: https://github.com/hovancik/stretchly.git
```

Comment:
this shouldn't be in a separate manifest since it looks like it's the main source.

Reply:
Removed.